product_docs/docs/biganimal/release/overview/03_security/02_reviewing_asc_recommendations.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: "Reviewing Microsoft Defender for Cloud recommendations"
```

Comment:
Why are these two separate pages? There are some recommendations that they can't fix, and some that they can but we won't do it for them - but they're ~ASC~ Defender for Cloud recommendations all the same, right?

Reply:
Merged the pages.